examples/notebooks/intro_notebook.ipynb

```diff
@@ -0,0 +1,307 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "Managing kubernetes objects using common resource operations with the python client\n",
+    "-----------------------------------------------------------------------------------------------\n",
+    "\n",
+    "Some of this operations include;\n",
+    "\n",
+    "- **`create_xxxx`** : create a resource object. Ex **`create_namespaced_pod`** and **`create_namespaced_deployment`**, for creation of pods and deployments respectively. This performs operations similar to `kubectl create`.\n",
```

Comment:
you don't write `kubectl create` in bold, but the other sentences you do. Try to be consistent.